h/services/user_delete.py

```diff
@@ -338,13 +362,15 @@ def update(self, model_class, select_stmnt, values: dict) -> list:
             .values(values)
             .returning(model_class.id)
         )
+
         if updated_ids:
             log.info(
-                "Updated %d rows from %s", len(updated_ids), model_class.__tablename__
+                f"{log_message}: %s", ", ".join(str(id_) for id_ in sorted(updated_ids))
```

Comment:
_If_ any rows were updated then log the given log message along with a sorted list of the row ids.